pages/1.11/installing/ent/custom/configuration/configuration-parameters/index.md

```diff
@@ -102,6 +103,17 @@ This topic provides configuration parameters available for [DC/OS Enterprise](ht
 | [zk_master_credentials](#zk-master)          | [enterprise type="inline" size="small" /] The ZooKeeper master credentials.  |
 | [zk_agent_credentials](#zk-agent)           | [enterprise type="inline" size="small" /] The ZooKeeper agent credentials.  |
 
+[enterprise]
+### adminrouter_auth_cache_enabled
+[/enterprise]
+
+_This option was added in DC/OS 1.11.1._
```

Comment:
1.11.1 is scheduled to ship on Apr 17. You might need to omit this info. (and add it later), if you plan to merge the PR now.